config/skim_Legacy2017_mib.cfg

```diff
@@ -3,7 +3,7 @@
 # do not select events with at least two jets in the final state
 # false : only >= 2 b jet ; true : all events
 ## for sync use beInclusive==true, for full porduction use false
-beInclusive = true
+beInclusive = false
```

Comment:
this should be true

Reply:
@camendola we used `beInclusive = true` when we were doing checks for HTT group, should we change it to `false` now?

Reply:
no, those events are needed for the tauID DM dependent, we can move it to false for the future rounds of ntuples

Reply:
that's true
@dzuole please fix this parameter

Reply:
@camendola @francescobrivio I took the liberty to add a comment here not to forget this in the future since I was close to the end of the skimming of the full 2017 when I received this comment :( :( :(